index.html

```diff
@@ -151,7 +163,7 @@ <h2>Contact me</h2>
             <input type="text" value="Mudasir">
             <input type="email" name="" id="" value="muda@nonihub.org">
             <textarea name="" id="" cols="30" rows="10" form="contact-form" placeholder="Write your message here"></textarea>
-            <input class="submit-btn" type="submit" value="Get In Touch">
+            <div class="btnholder"><input class="submit-btn" type="submit" value="Get In Touch"></div>
```

Comment:
- [ ] Kindly ensure that you add button states that are for interactivity. The states are eg on hover, active etc

![image](https://user-images.githubusercontent.com/86318284/192342551-07dacb01-6ae0-4ff2-a612-a7aa12a68ebe.png)